.gitignore

```diff
@@ -99,3 +99,4 @@ ENV/
 
```

Comment:
There are mixed feelings on this issue, I think it depends on what you agree on as a team, but I'll do some more research.